client/src/PageNotFound.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div class="center-grid">
+    <div class="center-grid__content">
+      <div class="text-center">
+        <img class="logo" src="./assets/CoolestProjectsLogo.png" />
+        <div class="not-found">
+          <h1>
+            It looks like you tried to go to a page that does not exist!
+          </h1>
+          <h1>
+            <a href="/">Please go to our homepage and try again</a>
```

Comment:
router-link to avoid a full reload ? cc @DanielBrierton

Reply:
Pretty sure Vue is clever enough to not cause a page refresh with this, but yeah, `<router-link>` would be best IMO.